rules/windows/execution_suspicious_short_program_name.toml

```diff
@@ -0,0 +1,27 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
```

Comment:
I think we're going to drop this to `development` and wait until EQL in the Endpoint or something else, because `length()` in Elasticsearch is nonperformant

Reply:
good to know